### DISCUSSION TITLE
필터링 AND 조건 OR 조건

### DISCUSSION THREAD

**Discussion by nan-noo** [woowacourse-teams/2022-moamoa#61] [토론]
지금은 모두 AND 조건으로 처리 중인데, 사실 같은 카테고리에 있는 조건을 고르면 OR 조건으로 처리하는 게 자연스럽지 않을까요?

예시)
카테고리: 기수, 언어, 분야 등
유저가 선택한 필터 조건: 4기, 3기, JavaScript

1. 현재 구현한 방법으로 필터링한 결과: `4기 AND 3기 AND JavaScript` -> 필터링 결과가 없음
2. 제안한 방법으로 필터링한 결과: `(4기 OR 3기) AND JavaScript` -> 4기와 3기 중 JavaScript에 해당하는 필터링 결과가 나옴

어떻게 생각하나요??

**Comment by tco0427** [comment]
동의합니다.😄
`기수`, `영역(be,fe)`, `스터디` 카테고리 별로 나눠서 카테고리 내에서는 OR 조건이 되도록 하면 될 것 같습니다!

**Comment by sc0116** [comment]
카테고리로 나누어서 하면 더 좋을 것 같아요 오홍홍~^^ 😊

**Reply by tco0427** [reply]
호호홍
